query.js

```diff
@@ -527,6 +529,8 @@ const projectForEntryList = entry => ({
     gender: entry.type,
     numTours: entry.numTours,
     entryLength: entry.biography.length + (entry.tours ? entry.tours.length : 0) + (entry.narrative ? entry.narrative.length : 0) + (entry.notes ? entry.notes.length : 0),
```

Comment:
i would be in favor to add the word count to the data sheets

Reply:
It might be faster, but since the "slow" option is not unbearably slow, I think the main consideration is whether we want it to be exported on the data sheets.  @ryanctan